### PR TITLE
Fixed DNS zone import issue with semicolon symbol

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
The content to be added to Changelog is as follows:
* Fixed issue https://github.com/Azure/azure-xplat-cli/issues/2869

### CODE REVIEW

File: lib/commands/arm/network/recordSetUtils._js [hunk 1]
```diff
@@ -377,4 +389,29 @@ function mergePTR(rs1, rs2) {
...
     }
   }
+}
+
+function sliceTXTValue(txt) {
+  var maxLength = constants.dnsZone.txtStringMaxLength;
+  var iterationCount;
+  var txtArray = [];
+
+  if (Object.prototype.toString.call(txt) === '[object Array]'){
```
Comment (amarzavery) [inline lib/commands/arm/network/recordSetUtils._js]: @MikhailTryakhov 
if you want to check whether something is an Array then there is a simple way to do it. `Array.isArray(txt)`. Please use that.

Comment (MikhailTryakhov) [inline lib/commands/arm/network/recordSetUtils._js]: wow. terribly sorry that forgot to rewrite this ugly line..

File: lib/commands/arm/network/recordSetUtils._js [hunk 2]
```diff
@@ -226,13 +226,25 @@ exports.getPropertyName = function (setType) {
...
 exports.splitTxtRecord = function(text) {
-  var maxLength = constants.dnsZone.txtStringMaxLength;
-  var iterationCount = text.length / maxLength;
-  var txtArray = [];
-  for(var index = 0; index < iterationCount; index++) {
-    txtArray.push(text.substr(index * maxLength, maxLength));
+  var encoded = text.replace(/\\"/g, '&quot;').replace(/\\/g, '');
+  var reg = /"[^"]*"/g;
+  var checkOnInvalidInput = encoded.replace(reg, '');
+  if (checkOnInvalidInput.length == encoded.length && !checkOnInvalidInput.trim().split(' ')[1]){
```
Comment (amarzavery) [inline lib/commands/arm/network/recordSetUtils._js]: === instead of ==

File: lib/commands/arm/network/dnsZone._js [hunk 3]
```diff
@@ -117,6 +117,7 @@ __.extend(DnsZone.prototype, {
           row.cell($('Name'), zone.name);
           var resInfo = resourceUtils.getResourceInformation(zone.id);
           row.cell($('Resource group'), resInfo.resourceGroup);
+          row.cell($('Tags'),tagUtils.getTagsInfo(zone.tags));
```
Comment (amarzavery) [inline lib/commands/arm/network/dnsZone._js]: shouldnt you be doing the same thing over here as well
`tagUtils.getTagsInfo(zone.tags) || '');`

**Pull request merged**